### ISSUE TITLE
Untranslated in-app notifications

### ISSUE THREAD

**Issue by snowteamer** [okTurtles/group-income#2897]
### Problem

Notifications from the bell icon popup don't appear to be translated:

<img width="497" height="416" alt="Image" src="https://github.com/user-attachments/assets/ee7da591-3ac2-4f92-a9b7-71ef28ceb1fc" />

### Solution

Not sure how to fix this.

I've been trying to debug it, but haven't found the cause yet.
- It doesn't seem to be caused by missing translations, since using `L()` directly in the console correctly translates a string from `templates.js`, despite manually emitting a notification producing an object with untranslated `bodyText` and `plainBodyText` fields:

<img width="616" height="413" alt="Image" src="https://github.com/user-attachments/assets/be38d57d-6752-4507-8984-e63b50845964" />

All this despite `L()` being apparently correctly used as necessary in all these scnearios in `notifications/templates.js`:

<img width="1266" height="597" alt="Image" src="https://github.com/user-attachments/assets/95c8ca85-7ee9-422e-b0eb-943c6fae2414" />
